src/Uniphar.Dotnet.Cli/Commands/GetTokenCommand.cs

```diff
@@ -11,23 +11,42 @@ public sealed class Settings : CommandSettings
         public required Guid ClientId { get; set; }
 
         [CommandOption("-s|--scope")]
-        [DefaultValue(".default")]
+        [DefaultValue(null)]
         public string? Scope { get; set; }
+
+        [CommandOption("--useCIAM")]
+        [DefaultValue("false")]
+        public bool? UseCIAM { get; set; }
     }
 
 
     public async override Task<int> ExecuteAsync(CommandContext context, Settings settings)
     {
-        var app = PublicClientApplicationBuilder.Create(settings.ClientId.ToString())
-                                               .WithTenantId(settings.TenantId.ToString())
-                                               .WithDefaultRedirectUri()
-                                               .Build();
+        var tenantId = settings.TenantId.ToString();
+        var clientId = settings.ClientId.ToString();
+        var authorityUri = (settings.UseCIAM ?? false) ? $"https://{tenantId}.ciamlogin.com" : $"https://login.microsoftonline.com/{tenantId}";
+
+        var app = PublicClientApplicationBuilder.Create(clientId)
+                                                .WithTenantId(tenantId)
+                                                .WithAuthority(authorityUri)
+                                                .WithDefaultRedirectUri()
+                                                .Build();
         
-        var scope = string.IsNullOrWhiteSpace(settings.Scope) ? ".default" : settings.Scope;
+        var scope = string.IsNullOrWhiteSpace(settings.Scope) ? $"{clientId}/.default" : settings.Scope;
 
-        var result = await app.AcquireTokenInteractive([$"{settings.ClientId}/{scope}"]).ExecuteAsync();
-        
-        AnsiConsole.MarkupLine($"{Environment.NewLine}[purple]{result.AccessToken}[/]");
+        AnsiConsole.MarkupLine($"[bold][steelblue1]- TenantId    : [/][/][italic]{tenantId}[/]");
+        AnsiConsole.MarkupLine($"[bold][steelblue1]- ClientId    : [/][/][italic]{clientId}[/]");
+        AnsiConsole.MarkupLine($"[bold][steelblue1]- AuthorityId : [/][/][italic]{authorityUri}[/]");
+        AnsiConsole.MarkupLine($"[bold][steelblue1]- Scope       : [/][/][italic]{scope}[/]");
+
+
+        var result = await app.AcquireTokenInteractive([scope]).ExecuteAsync();
+
+        //Access token is long and MarkupLine breaks the output in multiple lines with new line character
+        var color = Console.ForegroundColor;
+        Console.ForegroundColor = ConsoleColor.DarkGreen;
+        Console.WriteLine($"{Environment.NewLine}{result.AccessToken}");
```

Comment:
Might suggest pulling the extra new line out as it's own empty `AnsiConsole.WriteLine();` call, just to reinforce that we're purposefully adding an extra line break before the access token.